src/uct/ib/base/ib_device.c

```diff
@@ -224,7 +224,7 @@ ucs_status_t uct_ib_device_init(uct_ib_device_t *dev,
     /* Open verbs context */
     dev->ibv_context = ibv_open_device(ibv_device);
     if (dev->ibv_context == NULL) {
-        ucs_error("ibv_open_device(%s) failed: %m", ibv_get_device_name(ibv_device));
+        ucs_warn("ibv_open_device(%s) failed: %m", ibv_get_device_name(ibv_device));
```

Comment:
looking at the code, I think it's better to keep it as ucs_error(), otherwise UCT layer would not be consistent (can print warning and still return failure)